packages/now-cli/src/commands/env/ls.ts

```diff
@@ -47,15 +50,36 @@ export default async function ls(
 
   const lsStamp = stamp();
 
-  const records = await getEnvVariables(output, client, project.id, envTarget);
+  if (typeof nextTimestamp !== 'undefined' && Number.isNaN(nextTimestamp)) {
+    output.error('Please provide a number for flag --next');
+    return 1;
+  }
+
+  const data = await getEnvVariables(
+    output,
+    client,
+    project.id,
+    5,
+    envTarget,
+    nextTimestamp
+  );
+  const { envs: records, pagination } = data;
   output.log(
-    `${plural(
-      'Environment Variable',
-      records.length,
-      true
-    )} found in Project ${chalk.bold(project.name)} ${chalk.gray(lsStamp())}`
+    `${
+      records.length > 0 ? 'Environment Variables' : 'No Environment Variables'
+    } found in Project ${chalk.bold(project.name)} ${chalk.gray(lsStamp())}`
```

Comment:
This seems wrong because it in the case there is one env var, it will print `1 Environment Variables` as plural when it should be singular.

Why is this line being changed at all?

Reply:
Oh I understand now, it no longer prints the number of env vars 👍